docs/cli/api.md

```diff
@@ -476,7 +476,8 @@ grid run [OPTIONS] [RUN_COMMAND]...
 | `--description` | text | Run description; useful for note-keeping | None |
 | `--cluster` | text | N/A | `prod-2` |
 | `--strategy` | choice (`grid_search` &#x7C; `random_search`) | Hyper-parameter search strategy | None |
-| `--trials` | integer | Number of trials to run hyper parameter search | None |
+| `--num_trials` | integer | Number of samples from full search space that are used by the random_search strategy | None |
```

Comment:
these need to be updated in the cli code as well. @filintod is that done or should i open a PR?

Reply:
nvm i see it in there :)